application/models/AccessoryEntity.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
```

Comment:
variables, getters and setters for this.

Reply:
Alright